callout/src/main/java/com/google/apigee/ProtoMessageBuilders/ExecutionProtoMessageBuilder.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.apigee.ProtoMessageBuilders;
+
+import com.apigee.flow.execution.ExecutionContext;
+import com.apigee.flow.message.MessageContext;
+import com.google.apigee.Execute;
+
+/**
+ * Builder for Execution Protocol Buffer Message which handles the construction of the
+ * ExecutionContext and MessageContext Protocol Buffer Messages.
+ */
+public class ExecutionProtoMessageBuilder {
+
+  private final ExecutionContextProtoMessageBuilder executionContextProtoMessageBuilder;
+  private final MessageContextProtoMessageBuilder messageContextProtoMessageBuilder;
+
+  /**
+   * @param executionContextProtoMessageBuilder Builder for ExecutionContext Protocol Buffer
+   *     Messages.
+   * @param messageContextProtoMessageBuilder Builder for MessageContext Protocol Buffer Messages.
+   */
+  public ExecutionProtoMessageBuilder(
```

Comment:
What are your thoughts on making the build methods of `ExecutionContextProtoMessageBuilder` and `MessageContextProtoMessageBuilder` static, since each instance is the same (no instance variables in these classes)? Then this constructor isn't needed.

Reply:
makes sense. changed